package.json

```diff
@@ -5,28 +5,32 @@
   "main": "index.js",
   "scripts": {
     "eslint": "eslint .",
-    "test": "npm run eslint"
+    "test": "npm run eslint",
+    "prettier": "prettier --config .prettierrc '**/*.{js,json}'"
   },
   "repository": "blogfoster/eslint-config-blogfoster",
-  "keywords": [
-    "eslint"
-  ],
+  "keywords": ["eslint"],
   "author": "blogfoster GmbH",
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/blogfoster/eslint-config-blogfoster/issues"
   },
   "homepage": "https://github.com/blogfoster/eslint-config-blogfoster#readme",
   "dependencies": {
-    "babel-eslint": "^7.2.3",
-    "eslint-config-airbnb": "^15.0.1",
-    "eslint-plugin-import": "^2.3.0",
-    "eslint-plugin-jsx-a11y": "^5.0.3",
-    "eslint-plugin-mocha": "^4.9.0",
-    "eslint-plugin-react": "^7.1.0",
-    "estraverse-fb": "^1.3.1"
+    "babel-eslint": "^8.0.0",
+    "eslint": "^4.7.2",
```

Comment:
i guess this `eslint` from dependencies can be removed

Reply:
was also a peer dependency

Reply:
@cangoektas sorry i dont get it really. now we have eslint as `dependency`, `devDependency` and `peerDependency`. aren't the last two enough?

Reply:
@cangoektas if you check for example the the [airbnb config](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/package.json), you see they also just have eslint as dev and peer dependency but not as actual dependency

Reply:
Yeah, I guess you don't need it as a dep

Reply:
Only dev dep, since we lint this repo itself

Reply:
huh, you're right I ran it again without it and didn't get the peer dependency warning. Yes, `dev` + `peer` is enough